public class ExceptionHandlingDemo{
    public static void main(String[] args){
        int val = myMethod();
        System.out.println(val);
    }
    public static int myMethod(){
        try{
            return 112;
            
        }finally{
            System.out.println("This is Finally block");
            System.out.println("Finally block ran even after return atatement");
        }
    }
}
